Registry/Registry.EM_COMMON

```diff
@@ -2480,7 +2496,7 @@ rconfig   integer imomentum               namelist,physics      1             0
 rconfig   integer ishallow                namelist,physics      1             0       rh    "ishallow"  "shallow convection in G3 scheme"      ""
 rconfig   real    convtrans_avglen_m      namelist,physics      1             30      rh    "convtrans_avglen_m"  "averaging time for convective transport output variables (minutes)"      ""
 rconfig   integer num_land_cat            namelist,physics	1            21       -      "num_land_cat"                  ""      ""
-rconfig   integer use_wudapt_lcz          namelist,physics   1             0       -      "use_wudapt_lcz"                  ""      ""
+rconfig   integer use_wudapt_lcz          namelist,physics      1             0       -      "use_wudapt_lcz"                ""      ""
```

Comment:
white space change

Reply:
Another example of trying to line the columns up. We could choose to go with the majority of lines above or majority of lines below